iac/main/resources/manual-reference-data-ingestion.yml

```diff
@@ -326,3 +326,75 @@ ManualReferenceDataUploadRole:
             - Effect: Allow
               Action: s3:PutObject
               Resource: !Sub arn:aws:s3:::${Environment}-dap-raw-layer/reference-data/*
+
+ProcessReferenceDatAlertLogGroup:
+  Type: AWS::Logs::LogGroup
+  Properties:
+    LogGroupName: /aws/events/reference-data-ingestion-pipeline-alerts
+    KmsKeyId: !GetAtt KmsKey.Arn
+    RetentionInDays: 30
+
+ReferenceDataIngestionAlertingEventRule:
+  Type: AWS::Events::Rule
+  Properties:
+    Description: 'Reference Data Ingestion Alerting EventRule'
+    EventPattern:
+      source:
+        - reference-data-ingestion-pipeline
+    State: 'ENABLED'
+    Targets:
+      - Id: 'SendToCloudWatchLog'
+        Arn: !GetAtt ProcessReferenceDatAlertLogGroup.Arn
+      - Arn: !Ref SNSReferenceDataAlertTopic
+        Id: 'SendToSnsTopic'
```

Comment:
really minor but a tiny bit confusing that the first target has the dash with Id but the second one with Arn

Reply:
changed the order